dashboard/test/controllers/concerns/pd/workshop_filters_test.rb

```diff
@@ -8,11 +8,11 @@ class FakeController < ::ApplicationController
   end
 
   setup do
-    @params = {}
+    @params = ActionController::Parameters.new
     FakeController.any_instance.stubs(params: @params)
 
     @user = mock
-    @user.expects(admin?: true)
+    @user.stubs(admin?: true)
```

Comment:
You can always just update this attribute instead of stub it. Doesn't really matter